modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/WebController.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.modelservice.web;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author kashah
+ *
+ */
+@RestController
+public class WebController {
```

Comment:
This is a poor name as all controllers here will be web controllers. Choose a name that clarifies the responsibility this web controller has relative to others. For example this could be `HealthController` or similar.